src/main/java/gameframework/drawing/GameCanvasDefaultImpl.java

```diff
@@ -36,8 +39,11 @@ public void drawFullSizeImage(Graphics graphics, Image image) {
 	}
 
 	@Override
-	public void drawFullSizeImage(Image image) {
-		canvas.getGraphics().drawImage(image, 0, 0, canvas.getWidth(), canvas.getHeight(), canvas);		
+	public void drawFullSizeImage(Image image, int x, int y) {
```

Comment:
IIRC, `drawFullSizeImage` is supposed to draw an image that takes the full size of the canvas. Why would someone want to draw a "full-size" image only on a portion of the canvas?